web_timeline/static/src/js/web_timeline.js

```diff
@@ -172,25 +165,27 @@ odoo.define('web_timeline.TimelineView', function (require) {
                 onRemove: self.on_remove,
                 zoomKey: this.zoomKey
             };
-            if (this.default_window) {
-                var start = new moment();
-                var end;
-                switch (this.default_window) {
+            if (this.mode) {
+                var start, end = false;
```

Comment:
Any reason why `start` defaults to `undefined` and `end` defaults to `false`?

Reply:
No particular reasons, I often declare "null" variables like that for readability.
I will change it with :
```javascript
var start = false, end = false;
```
To be more explicit

Reply:
`undefined` is a falsey value, so if it doesn't really matter, just do `var start, end;`